src/app/zap-templates/zcl/data-model/chip/commodity-metering-cluster.xml

```diff
@@ -39,6 +39,7 @@ Git: 0.9.2-summer2025-228-g4ec7ea4c4
     <globalAttribute code="0xFFFD" side="either" value="1"/>
     <attribute code="0x0000" side="server" name="MeteredQuantity" define="METERED_QUANTITY" type="array" entryType="MeteredQuantityStruct" isNullable="true"/>
     <attribute code="0x0001" side="server" name="MeteredQuantityTimestamp" define="METERED_QUANTITY_TIMESTAMP" type="epoch_s" isNullable="true"/>
-    <attribute code="0x0002" side="server" name="TariffUnit" define="TARIFF_UNIT" type="TariffUnitEnum" isNullable="true" max="0x01"/>
+    <attribute code="0x0002" side="server" name="MeasurementType" define="TARIFF_UNIT" type="MeasurementTypeEnum" isNullable="true" max="0x0011" min="0x00"/>
```

Comment:
```suggestion
    <attribute code="0x0002" side="server" name="MeasurementType" define="MEASUREMENT_TYPE" type="MeasurementTypeEnum" isNullable="true" max="0x0011" min="0x00"/>
```
since we are changing this attribute too...